sql-jdbc/build.gradle

```diff
@@ -32,7 +32,7 @@ plugins {
 group 'com.amazon.opendistroforelasticsearch.client'
 
 // keep version in sync with version in Driver source
-version '1.10.0.0'
+version '1.10.1.0'
```

Comment:
Another version file to change? https://github.com/opendistro-for-elasticsearch/sql/blob/5ab1bc8f051c674b963b22b04979791256caf119/sql-jdbc/src/main/java/com/amazon/opendistroforelasticsearch/jdbc/internal/Version.java#L22